agents/blueprints/profile_extractor_agent.py

```diff
@@ -15,10 +15,9 @@ def profile_extractor(pipeline_kwargs: dict = {}):
     extractor.add_component("builder", PromptBuilder(template))
     extractor.add_component(
         "generator",
-        OllamaGenerator(
-            model=os.getenv("LLM_MODEL", "gemma"),
-            url=str(os.environ["LLM_URL"]),
-            timeout=int(os.getenv("LLM_TIMEOUT_SECONDS", 10 * 60)),
+        OpenAIGenerator(
+            model=os.getenv("LLM_MODEL", "gpt-3.5-turbo"),
+            # timeout=int(os.getenv("LLM_TIMEOUT_SECONDS", 10 * 60)),
```

Comment:
Is no timeout expected?

Reply:
No, openaigenerator does not accept timeout